docs/develop/astraeus/getting-started.md

```
@@ -0,0 +1,413 @@
+---
+sidebar_position: 1
+---
+
+# Getting Started
+
+This guide will help you set up the Astraeus API server on the Suave Toliman Testnet to easily use Transferable Accounts (TA). You will learn how to create, approve, and transfer TAs using the API server. Additionally, you will learn how to sign transactions on external chains using TAs.
+
+### Prerequisites
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)